source/docs/flow_control.md

```diff
@@ -80,9 +80,11 @@ the connection manager (`Envoy::Http::ConnectionManagerImpl`).  The router is
 responsible for intercepting watermark events for its own buffers, the individual upstream streams
 (if codec buffers fill up) and the upstream connection (if the network buffer fills up).  It passes
 any events to the connection manager, which has the ability to call `readDisable()` to enable and
-disable further data from downstream.
-
-TODO(alyssawilk) document the reverse path.
+disable further data from downstream.  On the reverse path, when the downstream connection
```

Comment:
Great docs! I read these before starting the review and they provided everything I was after.